templates/emails/hostedCollectives.otherPlans.limit.reached.hbs

```diff
@@ -0,0 +1,34 @@
+Subject: Plan reaches Limit
+
+<style>
+  .button {
+    background: linear-gradient(rgb(24, 105, 245) 0%, rgb(22, 89, 225) 100%);
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 18px;
+    color: rgb(255, 255, 255);
+    border-radius: 100px;
+    padding: 10px 16px;
+  }
+</style>
+
+{{> header}}
+
+<p>Hello,</p>
+
+<p>
+  We hope you are well! {{name}}, has reached the maximum number of Collectives for your plan on Open Collective. If you want to host more Collectives, please upgrade your plan. Please see below our plan options.
```

Comment:
@piamancini It's weird saying "{name}" then saying "for your plan", can you double check?

Reply:
ah sorry, my mistake. make it: {name} has reached the maximum number of collectives on the current plan. 

(please note, there's no comma after {name}